packages/webapp/src/views/credits-purchase/credits-confirmation.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'credits-form-layout.html' %}
+
+{% set hideBackLink = true %}
+{% set pageHeading = "Application complete" %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title govuk-!-margin-bottom-4">
+        {{ pageHeading }}
+      </h1>
+
+      <div class="govuk-panel__body govuk-!-font-size-27">
+        Your BNG credit reference<br> <strong data-testid="application-reference-value">{{ applicationReference }}</strong>
+      </div>
+    </div>
+
+    <p>We will send you a confirmation email.</p>
```

Comment:
Doesn't match latest copydeck. Should be `We have sent you a confirmation email.`